db/migrate/20201007213224_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.0]
```

Comment:
Why do you define change? And not index. You should remove text from the index as only the title and teaser will be displayed on the landing page.

Reply:
What do you mean? We never created this, it was automatically generate from the migrate?